model/src/main/java/io/paradoxical/cassieq/model/auth/AuthorizationLevel.java

```diff
@@ -0,0 +1,72 @@
+package io.paradoxical.cassieq.model.auth;
+
+import com.google.common.base.Joiner;
+import com.google.common.base.Splitter;
+import com.google.common.collect.ImmutableSet;
+import lombok.Getter;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.EnumSet;
+
+import static java.util.stream.Collectors.toList;
+
+public enum AuthorizationLevel {
+    None("n"),
+    ReadMessage("r"),
+    PutMessage("p"),
+    UpdateMessage("u"),
+    AckMessage("a"),
+    GetQueueInformation("g"),
+    CreateQueue("c"),
+    DeleteQueue("d");
+
+    public static final EnumSet<AuthorizationLevel> All =
+            EnumSet.of(ReadMessage, PutMessage, UpdateMessage, AckMessage);
+
+    public String Role = "BLAH";
+
+    @Getter
+    private final String shortForm;
+
+    AuthorizationLevel(final String shortForm) {
```

Comment:
Is the short form an aggregation of characters? Like "gcd" is "get, create, delete"?